pygmt/encodings.py

```diff
@@ -22,8 +22,11 @@
 - Adobe Symbol: https://en.wikipedia.org/wiki/Symbol_(typeface)
 - Zapf Dingbats: https://en.wikipedia.org/wiki/Zapf_Dingbats
```

Comment:
Hm. This wikipedia article uses a white space in the title, but everywhere else in this PR we use no white space.

```suggestion
- ZapfDingbats: https://en.wikipedia.org/wiki/Zapf_Dingbats
```